README.md

```diff
@@ -213,24 +213,32 @@ jobs:
 
 ### Docker image
 
-You can run tests in a GH Action in your Docker container.
+You can run the action in a Docker container.
 
 ```yml
-name: E2E in custom container
+name: Test in Docker
 on: push
 jobs:
   cypress-run:
     runs-on: ubuntu-22.04
-    # Cypress Docker image with Chrome v106
-    # and Firefox v106 pre-installed
-    container: cypress/browsers:node18.12.0-chrome106-ff106
+    # Cypress Docker image from https://hub.docker.com/r/cypress
+    # with browsers pre-installed
+    container:
+      image: cypress/browsers:latest
+      options: --user 1001
     steps:
       - uses: actions/checkout@v3
       - uses: cypress-io/github-action@v5
         with:
           browser: chrome
 ```
 
+Replace the `latest` tag with a specific version image tag from [`cypress/browsers` on Docker Hub](https://hub.docker.com/r/cypress/browsers/tags) to avoid breaking changes when new images are released (especially when they include new major versions of Node.js).
+
+Include `options: --user 1001` to avoid permissions issues.
```

Comment:
@nagash77 
Thanks for your question!

For `firefox` the parameter `--user 1001` must be added. For other browsers (`chrome`, `edge` and `electron`) it is optional and safe to use. I was able to test this on GitHub-hosted runners.

I would prefer for a fuller explanation to be added to the https://github.com/cypress-io/cypress-docker-images README where it belongs, however we have been unsuccessful so far in getting the related PR https://github.com/cypress-io/cypress-docker-images/pull/900, which is assigned to @cacieprins, moved along. That would have added an explanation to the Docker README.

The full explanation relates to permissions on the Cypress Docker images, the way that Mozilla Firefox is installed into Docker, and the fact that Firefox blocks running as root user due to their security considerations (see https://bugzilla.mozilla.org/show_bug.cgi?id=1460297). This is too much information to add here though.

Unfortunately I'm unclear on the implications for GitHub self-hosted runners. Various users have raised issues for self-hosted runner but then not followed through on investigations. I don't have a self-hosted runner to test with. All I can say is that I don't know of any proven case on a self-hosted runner where `--user 1001` does **not** work.

I could change the text to:

"You must include `options: --user 1001` if you are testing with the `firefox` browser. For other browsers it is optional."

Let me know if you think that would resolve your concerns.

Reply:
I just reviewed and merged the PR explaining this problem a bit more in depth in the README in the cypress-docker-images repo.  I'll go ahead and get this PR merged as well.

Reply:
@nagash77 

Thanks very much!

Thanks to your merge, the Docker README now explains the problem in depth and since this README in the Cypress GitHub Actions repo links to the Docker README then users should be able to find the reasons behind it all.